### DISCUSSION TITLE
Add a link to docs of the specific token type.

### DISCUSSION THREAD

**Discussion by jtagcat** [thinkst/canarytokens#246] [Feature Requests]
![image](https://user-images.githubusercontent.com/38327267/132960013-661c65bd-38fc-474f-8a71-ec010e5fc4bf.png)

It'd be neet, if there was 'Docs ![open externally](https://design.firefox.com/icons/icons/desktop/open-in-new-16.svg)' to the right of the selection, linking to (in this case) https://docs.canarytokens.org/guide/http-token.html

**Comment by acharyasourav** [comment]
Hey @jtagcat can you please clarify I don't completely understand what you mean. I would like to work on this.

**Comment by jtagcat** [comment]
![image](https://user-images.githubusercontent.com/38327267/190617071-e55a17ac-85b0-49a7-8080-825b1c4f06c3.png)

While this is selected from the drop-down, it should have a link to the docs. Either below the dropdown, or next to it, with a ![questionmark with circle, help](https://design.firefox.com/icons/icons/desktop/help-16.svg); in this case linking to https://docs.canarytokens.org/guide/http-token.html

While DNS is selected: 
![image](https://user-images.githubusercontent.com/38327267/190617702-9a9909c7-bd2d-4937-bce8-d19fa0195927.png)

It should have a link to https://docs.canarytokens.org/guide/dns-token.html

and so on.

**Comment by roberto-aldera** [comment]
Fixed in new UI.
